pkg/plugins/runtime/k8s/controllers/inbound_converter.go

```diff
@@ -118,15 +116,13 @@ func inboundForServiceless(zone string, pod *kube_core.Pod, name string, nodeLab
 		pod.Status.ContainerStatuses,
 		pod.Status.InitContainerStatuses,
 	); cs != nil && !cs.Ready {
-		state = mesh_proto.Dataplane_Networking_Inbound_NotReady
 		health.Ready = false
 	}
 
 	return &mesh_proto.Dataplane_Networking_Inbound{
-		Port:   mesh_proto.TCPPortReserved,
-		Tags:   tags,
-		State:  state,
-		Health: &health, // write health for backwards compatibility with Kuma 2.5 and older
```

Comment:
you wrote in pr description that you left health as is, but you removed if from here, is it correct?

Reply:
it's not 🤦